mappings/net/minecraft/util/registry/DynamicRegistryManager.mapping

```diff
@@ -45,6 +46,10 @@ CLASS net/minecraft/class_5455 net/minecraft/util/registry/DynamicRegistryManage
 		COMMENT
 		COMMENT @throws IllegalStateException if the registry does not exist
 		ARG 1 key
+	METHOD method_31060 register (Lcom/google/common/collect/ImmutableMap$Builder;Lnet/minecraft/class_5321;Lcom/mojang/serialization/Codec;Lcom/mojang/serialization/Codec;)V
+		ARG 0 infosBuilder
+		ARG 1 reegistryRef
```

Comment:
```suggestion
		ARG 1 registryRef
```

Reply:
Fixed this typo (the mapping was present upstream so I just used the upstream names there).